Documentation/CraftingNuGetPackages.md

```diff
@@ -0,0 +1,60 @@
+# Crafting a NuGet Package for MSB4U
+
+> MSBuildForUnity (MSB4U) consumes .NET based packages without any modifications to the package. This documentation describes how C++, WinRT or packages with Unity-specific contents can easily be adapted to work with MSBuildForUnity.
```

Comment:
It might be worth making this section stand out a bit more somehow. I missed it on my first readthrough